### PR TITLE
Move to go modules

### PR DESCRIPTION
**Pull request by ferhatelmas**
Updated for modules.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 language: go
-sudo: false
 go:
-  - '1.10'
+  - '1.13.x'
 
 before_install:
```
Comment (posener) [inline .travis.yml]: I think that this stage is no longer necessary...

Comment (ferhatelmas) [reply .travis.yml]: yes not needed, dropped.

It could be used for caching but for a small module, it's faster to install fresh.

**Comment by ferhatelmas** [comment]
@posener Could you please push a new tag for it, v1.2.0 for example?

**Comment by posener** [comment]
Done

**Comment by ferhatelmas** [comment]
Thank you very much :+1:

**Pull request merged**